modules/odc-creating-apiserversource.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * applications/application_life_cycle_management/odc-knative-event-sources.adoc
+
+[id="odc-creating-apiserversource_{context}"]
+= Creating an ApiServerSource event source using the Developer perspective
+
+.Procedure
+
+. Navigate to the **Add** page.
++
+image::event-sources-odc.png[Add an Event Source in the Developer perspective]
+. Select **Event Sources** -> **ApiServerSource**.
++
+image::apiserversource-odc.png[Create an ApiServerSource]
+. Configure the **ApiServerSource** settings:
++
+image::apiserver-odc-config-1.png[Configuring the ApiServerSource]
++
+.. Enter `sources.knative.dev/v1alpha1` as the **APIVERSION**, and `ApiServerSource` as the **KIND**.
```

Comment:
here `ApiVersion` and `kind` is not of ApiServerSource for any sources this will be handled internally. Here these are for resources needs to be watched it could be more then one pair there is option in UI to add more.

```
  resources:
  - apiVersion: v1
    kind: Event
```

https://github.com/knative/docs/blob/8dcca2764db60006bf256fb98f5ba32783b80ec5/docs/eventing/samples/kafka/channel/020-k8s-events.yaml#L9-L11

even Screenshot may need update.

@matzew can add more input

Reply:
@matzew can you clarify please, thanks.

Reply:
I don't understand what you mean. Clarify what ? 

Let me guess here.  Since the for is already representing the `ApiServerSource`, the fields capture what's supported for its `spec`.  So the two fields on the form (`apiVersion` and `kind`) are to capture the K8s type to be observed, eg:

```
  resources:
  - apiVersion: v1
    kind: Event
```

As in the above example the `v1` versioned `Event` object, representing k8s platform events

Reply:
@invincibleJai the fields are not to describe the source, right? Or am I getting that wrong?

It;s for the APi of the API_SERVERS_SOURCE, right ?

Reply:
> @invincibleJai the fields are not to describe the source, right? Or am I getting that wrong?
> 
> It;s for the APi of the API_SERVERS_SOURCE, right ?

yes fields here are not to describe `ApiVersion` and `kind` for apiServerSource but to resources i.e events as described above.